src/ActionModal/components/DelegationModal.vue

```diff
@@ -136,6 +137,21 @@ export default {
 
       return this.fromOptions[this.selectedIndex].address
     },
+    validatorMoniker() {
```

Comment:
Something about this function (and I've seen the code elsewhere), is we seem to be calculating something we should already know?